payment-handler/app-can-make-payment-false.js

```diff
@@ -0,0 +1,6 @@
+// Copyright © 2017 Chromium authors and World Wide Web Consortium, (Massachusetts Institute of Technology, ERCIM, Keio University, Beihang).
+
+// This app always responds to CanMakePaymentEvent with "false".
+self.addEventListener('canmakepayment', (canMakePaymentEvent) => {
+    canMakePaymentEvent.respondWith(false);
```

Comment:
Although this gets auto wrapped with a promise, it might be good to have two tests:

```JS
canMakePaymentEvent.respondWith(false);
```

And:

```JS
canMakePaymentEvent.respondWith(Promise.resolve(false));
```

Reply:
It might be good to also test for:

```JS
canMakePaymentEvent.respondWith(Promise.reject(new Error("Custom error")));
```

Reply:
Added more respondWith() patterns.